webpack/configs/dev-base.js

```diff
@@ -27,7 +27,6 @@ module.exports = merge.strategy( {
 	],
 	optimization: {
 		splitChunks,
-		noEmitOnErrors: true, // NoEmitOnErrorsPlugin
```

Comment:
This optimization setting has been deprecated in favor of the reverse, which defaults to the setting we want: `emitOnErrors: false`.